Argo/Globals/JSONDeserializer.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import Runes
+
+public struct JSONDeserializer {
+  public static func deserialize(data: NSData) -> JSON? {
```

Comment:
I just realized that we shouldn't be obscuring the ability to set reading options here. Consider a response that doesn't begin with an array or object. For instance...

```
response_date: 2015-03-04T12:43:20Z,
post: {
    id: 1
    title: "JSON Fragments Considered Harmful"
}
```

Attempting to parse this with `NSJSONSerialization` without specifying the `NSJSONReadingAllowFragments` option would return an error: `The operation couldn’t be completed. (Cocoa error 3840.)" (JSON text did not start with array or object and option to allow fragments not set.)`

To work around this, I suggest we overload `deserialize(data:) -> JSON?` with `deserialize(data:allowFragments:) -> JSON?`. 

I'm pretty sure we can disregard the other reading options, `.MutableContainers` and `.MutableLeaves` but I could be convinced that we should add more overloads or a function that just takes an `NSJSONReadingOptions` if there's a really good use case for it.